stepfunctions/src/main/scala/awscala/stepfunctions/StepFunctions.scala

```diff
@@ -0,0 +1,131 @@
+package awscala.stepfunctions
+
+import java.util
+import java.util.concurrent.TimeUnit
+
+import awscala._
+import com.amazonaws.ClientConfiguration
+import com.amazonaws.services.stepfunctions.model._
+import com.amazonaws.services.{ stepfunctions => aws }
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ blocking, ExecutionContext, Future }
+import scala.util.Try
+
+object StepFunctions {
+  private val defaultSocketTimeout = TimeUnit.SECONDS.toMillis(70).toInt
```

Comment:
I prefer naming as `DEFAULT_SOCKET_TIMEOUT` for this because it's a kind of static field.